packages/editor/src/components/post-preview-button/index.js

```diff
@@ -7,7 +7,16 @@ import { get } from 'lodash';
  * WordPress dependencies
```

Comment:
It is a bit weird that the UI to switch the device is in the "Editor" package but the logic to apply the simulated queries is in the "edit-post" package.

Reply:
I added the UI to the existing preview button, which lives in the Editor package. Does it still make sense to have this component in this package? I'm unsure of the reasons it was originally put here. 

Or, I could add this UI as a new component that could live in "edit-post", and import the existing preview button into it. That might be a better approach if we're thinking of adding in other types of preview buttons later.

Reply:
> I could add this UI as a new component that could live in "edit-post", and import the existing preview button into it. That might be a better approach if we're thinking of adding in other types of preview buttons later.

I think that might be better yes. To be fair, the "editor" package is not the easiest to understand, the reason for its existence is to allow third-party users to build alternative editor layouts (alternative edit-post packages)